### PR TITLE
Consolidate Pearlboards boards and add support for Pearl 40%

### PR DESCRIPTION
**Pull request by Koobaczech**
Moved older existing Pandora entry into a newly created home directory for my projects, and added new keyboard entry, Pearl.

## Description

Hello VIA team, sending you all well wishes on these upcoming holidays. I have first created a new directory for my boards, secondly moved an existing keyboard json file (pandora.json) into it, as well as added a new json file (pearl.json) in this directory. Thank you!

## QMK Pull Request 

https://github.com/qmk/qmk_firmware/pull/15005

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/pearlboards/pearl.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+          {
+            "label": "Audio Clicky Enable",
+            "type": "toggle",
+            "content": ["id_qmk_audio_clicky_enable", 4, 2]
+          }
+        ]
+      }
+    ]
+  }
+],
```
Comment (Xelus22) [inline v3/pearlboards/pearl.json:26]: I don't see any audio things in your QMK config?
https://github.com/qmk/qmk_firmware/tree/master/keyboards/pearlboards/pearl

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/pearlboards/pearl.json (hunk 1 above)
Comment (Koobaczech) [reply v3/pearlboards/pearl.json:26]: My friend, please take a look at the Pearl rules.mk file [here](https://github.com/qmk/qmk_firmware/blob/master/keyboards/pearlboards/pearl/rules.mk), and take mention of where it says 
**AUDIO_ENABLE = yes          # Audio output**

My best, Koob

Comment (Xelus22) [inline v3/pearlboards/pearl.json:26]: There already is a QMK audio for you. Can you test if this works?
```suggestion
  "menus": ["qmk_rgblight", "qmk_audio"],
```

**Review by Xelus22 — CHANGES_REQUESTED** [review]

**Comment by Koobaczech** [comment]
Suggested change worked great. Thank you my friend.

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**